src/Tribe/Repository.php

```diff
@@ -2126,7 +2126,7 @@ protected function build_meta_query( $meta_key, $meta_value = 'value', $compare
 				$args['meta_query'][ $array_key ]['value'] = $meta_value;
 			}
 
-			if ( 0 === strpos( $type_or_format, '%' ) ) {
+			if ( ! empty( $type_or_format ) && 0 === strpos( $type_or_format, '%' ) ) {
```

Comment:
```suggestion
			if ( is_string( $type_or_format ) && 0 === strpos( $type_or_format, '%' ) ) {
```
Probably better to check for string instead of the various types that would pass the empty check.

Reply:
Good point. I was thinking it would not pass `0` or an empty string to the next check (strpos) since that will fail the check anyway?